sw/device/examples/hello_world/hello_world.c

```diff
@@ -29,6 +30,12 @@ int main(int argc, char **argv) {
   LOG_INFO("Hello World!");
   LOG_INFO("Built at: " __DATE__ ", " __TIME__);
 
+  // End the test here for DV simulation, since the rest of the code is more
```

Comment:
I think that `hello_world` isn't really a test. Do we need to build it as a test?

Reply:
Great thanks!